frontend/pages/hosts/details/HostDetailsPage/HostDetailsPage.tsx

```diff
@@ -784,6 +801,7 @@ const HostDetailsPage = ({
         </div>
         <HostSummaryCard
           summaryData={summaryData}
+          osUpdatesData={getOSUpdatesFromMDMConfig(host.platform)}
```

Comment:
Nit, can we call this prop the same thing we call the corresponding type, so maybe `osUpdateRequirement`

Reply:
Yeah, I chose naming closer to the API response, but specifying these are version requirements in the codebase is definitely easier to read. 

```
macos_updates: IAppleDeviceUpdates;
ios_updates: IAppleDeviceUpdates;
ipados_updates: IAppleDeviceUpdates
```

Reply:
Renaming to `osVersionRequirement`